app.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "collections",
+  "repository": "https://github.com/alphagov/collections",
+  "env": {
+    "GOVUK_APP_DOMAIN": {
+      "value": "www.gov.uk"
+    },
+    "GOVUK_ASSET_ROOT": {
+      "value": "https://govuk-collections.herokuapp.com"
```

Comment:
I think you're right, but this is only the initial configuration. I think I need to change the Procfile in order to override this value:

https://github.com/alphagov/government-frontend/blob/master/Procfile#L1